addons/l10n_sa_edi/tests/test_account_move.py

```diff
@@ -0,0 +1,47 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+
+from .common import TestSaEdiCommon
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install_l10n', '-at_install', 'post_install')
+class TestAccountMove(TestSaEdiCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='sa', edi_format_ref='l10n_sa_edi.edi_sa_zatca'):
+        super().setUpClass(chart_template_ref=chart_template_ref, edi_format_ref=edi_format_ref)
+
+        cls._setup_branches()
+
+    @classmethod
+    def _setup_branches(cls):
+        cls.sa_branch = cls.env['res.company'].create({
+            'name': 'SA Branch',
+            'parent_id': cls.company.id,
+            'country_id': cls.company.country_id.id,
+        })
+
+    def testInvoiceWithMistmatchedCompanies(self):
```

Comment:
I know the other file test is not consistent, but at least for your new test, don't use camel cases, use the Python format